packages/server/bin/ts-www

```diff
@@ -7,6 +7,12 @@
  */
 
 require('ts-node/register')
+
+//need to initialize before everything else
+//knex is a singleton controlled by module so can't wait til app init
+const { initOpenTelemetry } = require('../dist/otel')
```

Comment:
maybe this can be done in app.ts or bootstrap.js to avoid duplication?

Reply:
it has be done before knex is created.  I guess I could put it at the top of app.ts.  Can talk about it.